README.md

```diff
@@ -9,41 +9,61 @@ So to launch the server, it will only start the server.
 Mods and bots are managed at startup by checking your `config.yml`.
 `npm` is only invoked if changes are made to your `config.yml`.
 
-## Usage
+## Installation (Recommended)
+**Step 1: Clone/Download this repository.**
 
-The recommended way to use this image is through docker compose.
-An example setup with [screepsmod-mongo] can be seen in the compose file [docker-compose.yml](docker-compose.yml) in this repo.
-This spins up a server with a mongo and redis service beside it.
-Remember that `screepsmod-mongo` must be in the list of mods in your `config.yml`
+**Step 2:  Download Docker Desktop** 
 
-Copy the compose file and run `docker compose up`. You should see the services starting.
-You can now access your private screeps server on `http://localhost:21025`
+- [Download For Mac (Intel)](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=dd-smartbutton&utm_location=module) 
+- [Download For Mac (Apple Silicon)](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=dd-smartbutton&utm_location=module)
+- [Download For Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=dd-smartbutton&utm_location=module)
+- [Download For Linux](https://docs.docker.com/desktop/linux/install/)
 
-A lighter alternative without mongo and redis could look like this
-```yml
-version: '3'
-services:
-  screeps:
-    image: jomik/screeps-server:edge
-    volumes:
-      - ./config.yml:/screeps/config.yml
-      - screeps-data:/data
-    ports:
-      - 21025:21025/tcp
-    environment:
-      STEAM_KEY: ${STEAM_KEY:?"Missing steam key"}
-    restart: unless-stopped
-
-volumes:
-  screeps-data:
+**Step 3: Open your download directory `.../screeps-server/`** 
 
-```
+**Step 4: Edit `config.yml`**
 
-To access the CLI you can use `docker compose exec screeps cli`
+- Add your username to the `whitelist:`
+- Adjust other settings to your likings.
+	- [Docs: screepsmod-auth](https://github.com/ScreepsMods/screepsmod-auth)
+	- [Docs: screepsmod-admin-utils](https://github.com/ScreepsMods/screepsmod-admin-utils)
```

Comment:
This actually is not currently supported - I have not found a need for it.
If we want this, then we need to decide on the best way to expose this ability :)